CHANGELOG.rst

```diff
@@ -8,6 +8,7 @@ Changelog
 
 .. note:: This version is not yet released and is under active development.
 
+* Added class methods for Poly1305 sign and verify operations.
```

Comment:
Please make these links to the methods in the docs.

Reply:
Good catch! I've updated the Changelog to link to the docs.